pkg/content/element/element.go

```diff
@@ -0,0 +1,260 @@
+package element
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/readium/go-toolkit/pkg/manifest"
+)
+
+// Note: We can't embed structs/interfaces in the interfaces otherwise they become
+// "non-basic", meaning we then can't use them as returns for other interfaces like
+// [Iterator], where it's the return type of many of the functions. Maybe we should
+// rethink this approach with all the interfaces later when not copying the kotlin.
+
+// Represents a single semantic content element part of a publication.
+type Element interface {
+	// AttributesHolder
+	Language() string
+	AccessibilityLabel() string
+	Attribute(key AttributeKey) *Attribute[any]
+	Attributes(key AttributeKey) []Attribute[any]
```

Comment:
How about just `Attributes() AttributesHolder` to simplify the implementations?

Reply:
@mickael-menu You mean make a `type AttributesHolder []Attribute[any]`?

Reply:
There's one already here: https://github.com/readium/go-toolkit/blob/ba558c981288f460e690a1450b424a0b2ed8e1db/pkg/content/element/attributes.go#L22-L24